Data/Sector/Hong-Kong-Sector-File.ese

```diff
@@ -3601,19 +3605,21 @@ COPX:VHHH:25R:VEPIK:*:*:VHHK·VHHH_SCT_ATZ_N·000·020:VHHK·VHHK_SCT_25_DEP_02
 
 COPX:VHHH:*:OCEAN:RASSE:*:VHHK·VHHK_SCT_25_DEP_06·135·250:VHHK·VHHK_SCT_TRE_02·250·365:25000:*:OCEAN
 COPX:VHHH:*:OCEAN:SKATE:*:VHHK·VHHK_SCT_25_DEP_06·135·250:VHHK·VHHK_SCT_TRS_01·250·365:25000:*:OCEAN
-COPX:VHHH:*:LAKES:*:*:VHHK·VHHK_SCT_25_DEP_06·135·250:VHHK·VHHK_SCT_TRE_02·250·365:25000:*:LAKES
+COPX:VHHH:*:LAKES:*:*:VHHK·VHHK_SCT_25_DEP_06·135·250:VHHK·VHHK_SCT_TRE_02·000·250:23000:*:LAKES
+COPX:VHHH:*:LAKES:*:*:VHHK·VHHK_SCT_25_DEP_06·135·250:VHHK·VHHK_SCT_TRE_02·000·250:17000:*:LAKES
 COPX:VHHH:*:OCEAN:RASSE:*:VHHK·VHHK_SCT_07_DEP_07·000·250:VHHK·VHHK_SCT_TRE_02·250·365:25000:*:OCEAN
 COPX:VHHH:*:OCEAN:SKATE:*:VHHK·VHHK_SCT_07_DEP_07·000·250:VHHK·VHHK_SCT_TRS_01·250·365:25000:*:OCEAN
-COPX:VHHH:*:LAKES:*:*:VHHK·VHHK_SCT_07_DEP_07·000·250:VHHK·VHHK_SCT_TRE_02·250·365:25000:*:LAKES
+COPX:VHHH:*:LAKES:*:*:VHHK·VHHK_SCT_07_DEP_07·000·250:VHHK·VHHK_SCT_TRE_02·000·250:23000:*:LAKES
+COPX:VHHH:*:LAKES:*:*:VHHK·VHHK_SCT_07_DEP_07·000·250:VHHK·VHHK_SCT_TRE_02·000·250:27000:*:LAKES
```

Comment:
This should be 17000 let me fix that

Reply:
does not lgtm but once you fix that then lgtm